pom.xml

```diff
@@ -95,6 +95,12 @@
         <scope>import</scope>
         <type>pom</type>
       </dependency>
+      <dependency>
+        <groupId>org.hamcrest</groupId>
+        <artifactId>hamcrest-core</artifactId>
+        <version>2.2</version>
```

Comment:
Something is wrong here, this is supposed to be pinned by the parent POM. I suspect some other dependency is throwing things off. Oh well, can be fixed up some other time.